src/main/java/org/elasticsearch/index/query/SimpleQueryStringBuilder.java

```diff
@@ -114,24 +125,46 @@ public SimpleQueryStringBuilder flags(SimpleQueryStringFlag... flags) {
         this.flags = value;
         return this;
     }
+    
+    SimpleQueryStringBuilder flags(int flags) {
```

Comment:
Thanks for catching this - it's an artefact of how I generated the query in the test initially. Will remove.